spec/unit/7110_collectConfigurationNetworkInfo_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rspec'
+require_relative '../spec_helper'
+require_relative '../../core/session'
+require_relative '../../core/node_product'
+require_relative '../../core/out'
+require_relative '../../core/repo_manager'
+require_relative '../../core/exception_handler'
+require_relative '../../core/network'
+
+describe 'Network' do
+
+  before :all do
+    $out = Out.new
+    $session = Session.new
+    $session.isSilent = true
+    $session.mdbciDir = Dir.pwd
+    $exception_handler = ExceptionHandler.new
+    boxesPath = './BOXES'
+    $session.boxes = BoxesManager.new boxesPath
+    reposPath = './repo.d'
+    $session.configFile='instance.json'
+    $session.awsConfigFile='aws-config.yml'
+    $session.repos = RepoManager.new reposPath
+    $session.checkConfig
+    $session.loadCollections
+  end
+
+  # Before all tests must be generated configurations
+  # vagrant machine must be running
+  # for mdbci node must be created appropriate mdbci_template file and
+  # must be prepared box with IP and keyfile location that is targeting real running machine
+  # that can be accessed through ssh
+  
+  file_network_config = "#{ENV['configPath']}_network_config"
+  nil_file_network_config = "_network_config"
+  wrong_file_network_config = "WRONG_PATH_network_config"
+  
+  it 'collectConfigurationNetworkInfo should raise error: wrong path' do
+    lambda{collectConfigurationNetworkInfo(ENV['stoppedConfigPath'])}.should raise_error("Incorrect node")
+  end
+
+  it 'collectConfigurationNetworkInfo should return correct Hash' do
+    result = collectConfigurationNetworkInfo(ENV['configPath'])
+    result.each do |key,value|
+      case key
```

Comment:
Is the case when key does not match any of **when** patterns checked?
All case / switch constructions must include default  option.